CMake 4.3 refuses to process the project due to an unsupported version
being indicated.

Upstream changed the file to indicate 3.28 compatibility at some point
using this range specification.

--- a/libjpeg_turbo/CMakeLists.txt	2022-08-08 23:03:55.000000000 +0200
+++ b/libjpeg_turbo/CMakeLists.txt	2026-08-22 18:11:25.371262201 +0200
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 2.8.12...3.28)
 # When using CMake 3.4 and later, don't export symbols from executables unless
 # the CMAKE_ENABLE_EXPORTS variable is set.
 if(POLICY CMP0065)
